src/main/java/com/dreammaster/scripts/ScriptGregtechPlusPlus.java

```diff
@@ -191,6 +192,23 @@ public void loadRecipes() {
                 ItemList.Electric_Pump_MV.get(1L),
                 "circuitPrimitive");
 
+        // Shirabon and Eternity
+        GTValues.RA.stdBuilder()
+                .itemInputs(
+                        getModItem(EternalSingularity.ID, "combined_singularity", 1, 15, missing),
+                        ItemList.EnergisedTesseract.get(1),
+                        ItemUtils.getSimpleStack(GenericChem.TemporalHarmonyCatalyst, 0))
+                .itemOutputs(GTOreDictUnificator.get("dustShirabon", 64), ItemList.Timepiece.get(1))
+                .fluidInputs(MaterialsUEVplus.PrimordialMatter.getFluid(1152))
+                .fluidOutputs(MaterialsUEVplus.Eternity.getMolten(9216), MaterialsUEVplus.Time.getMolten(18432))
+                .metadata(QFT_FOCUS_TIER, 4).duration(20 * SECONDS).eut(TierEU.RECIPE_UMV)
+                .addTo(quantumForceTransformerRecipes);
+        addForestryRecipes();
```

Comment:
is this call surrounded with a if mod loaded check ? otherwise it will try to call a method that has been removed and it will fail with a method not found exception

Reply:
nice